test/helper.ts

```diff
@@ -49,9 +50,16 @@ export const spanWithParent: ReadableSpan = Object.assign(
+export const spanJSON =
```

Comment:
with real json it could be easier to keep it as json (easier to read and format), and then compare in tests without `JSON.stringify`